src/test/groovy/org/prebid/server/functional/tests/DebugSpec.groovy

```diff
@@ -304,4 +309,39 @@ class DebugSpec extends BaseSpec {
         null         || 0
         null         || null
     }
+
+    def "PBS should return HTTP call type when it's default bidder call"() {
+        given: "Default basic generic BidRequest"
+        def bidRequest = BidRequest.defaultBidRequest
+
+        when: "PBS processes auction request"
+        def response = defaultPbsService.sendAuctionRequest(bidRequest)
+
+        then: "Response should contain call type HTTP"
+        assert response.ext?.debug?.httpcalls[GENERIC.value].first().calltype == HTTP
```

Comment:
`calltype` should be in camelCase in the model.

Reply:
But in `ExtHttpCall.class` all fields are not in camelCase.

Reply:
And that is the error, unfortunately. Model should adhere to Java code conventions.